index.html

```diff
@@ -297,6 +297,146 @@ <h3>🌏 World Presidents</h3>
         </div>
         <div class="age">42 years</div>
       </div>
+      <!-- Country 21 -->
+      <div class="row">
+        <div class="num">21</div>
+        <div class="flag">🇧🇴</div>
+        <div class="country_Capital">Bolivia, Santa Cruz de la Sierra</div>
```

Comment:
Hey. Bolivia's Capital is Sucre. Santa Cruz is just the largest city

Reply:
Done